### PR TITLE
Package localhost-permission-allow-list.txt

### PR DESCRIPTION
**Pull request by ShivanKaul**
Merge only once https://github.com/brave/adblock-lists/pull/1156/ is merged.

### CODE REVIEW

**Review by antonok-edm — APPROVED** [review]
Seems reasonable. Make sure to test `npm run data-files-local-data-files` locally and check that the output in the `build` folder is what you expected.

File: scripts/packageComponent.js [hunk 1]
```diff
@@ -131,7 +132,9 @@ const getDATFileListByComponentType = (componentType) => {
     case 'local-data-files-updater':
       return [path.join('brave-lists', 'debounce.json'),
         path.join('brave-lists', 'clean-urls.json'),
-        path.join('brave-lists', 'https-upgrade-exceptions-list.txt')].concat(
+        path.join('brave-lists', 'https-upgrade-exceptions-list.txt'),
+        path.join('brave-lists', 'localhost-permission-allow-list.txt'),
```
Comment (antonok-edm) [inline scripts/packageComponent.js:136]: I like trailing commas, but unfortunately CI does not :stuck_out_tongue:

**Comment by ShivanKaul** [comment]
@antonok-edm 

> Seems reasonable. Make sure to test npm run data-files-local-data-files locally and check that the output in the build folder is what you expected.

Hm I don't get a `build/` folder, do you mean the `brave-lists/` folder?

**Comment by ShivanKaul** [comment]
Verified the new list shows up correctly after running `npm run package-local-data-files -- --local-run`

**Pull request merged**